crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/cucumber/StatelessSteps.java

```diff
@@ -0,0 +1,175 @@
+        var magCourtOutcome = data.get("magCourtOutcome");
+        if (magCourtOutcome != null ) {
+            requestData.setMagCourtOutcome(MagCourtOutcome.valueOf(magCourtOutcome));
+        }
+        if (Boolean.parseBoolean(data.get("hasAPartner"))) {
+            requestData.setPartner();
+        }
+        String freq_string = data.get("frequency");
+        if (freq_string != null) {
+            var frequency = Frequency.valueOf(freq_string);
+            var income = data.get("income");
+            if (income != null) {
+                requestData.addIncome(new FrequencyAmount(frequency, new BigDecimal(income)));
+            }
+            var outgoings = data.get("outgoings");
+            if (outgoings != null) {
+                requestData.addOutgoings(new FrequencyAmount(frequency, new BigDecimal(outgoings)));
+            }
+            var partnerIncome = data.get("partnerIncome");
+            if (partnerIncome != null) {
+                requestData.addIncome(new FrequencyAmount(frequency, new BigDecimal(partnerIncome)));
+            }
+            var partnerOut = data.get("partnerOutgoings");
+            if (partnerOut != null) {
+                requestData.addOutgoings(new FrequencyAmount(frequency, new BigDecimal(partnerOut)));
+            }
+        }
+
+        requestData.setRequestType(StatelessRequestType.BOTH);
+    }
+
+    @And("children with ages")
+    public void childrenWithAges(DataTable dataTable) {
+        List<Map<String, Integer>> listOfData = dataTable.asMaps(String.class, Integer.class);
+
+        var dependantChildren = listOfData.get(0).entrySet().stream()
+                .filter(child -> child.getValue() > 0).map(child -> {
+            var age = Arrays.stream(AgeRange.values())
+                    .filter(a -> child.getKey().equals(a.getValue()))
+                    .findFirst().get();
+
+            return new DependantChild(age, child.getValue());
+        }
+        ).toList();
+        requestData.setChildren(dependantChildren);
+    }
+
+    // lambdas to check initial result
+    static Map<String, Consumer<Pair<String, StatelessInitialResult>>> initialCheckers = Map.of(
+            "imaResult", result_pair -> {
+                assertEquals(InitAssessmentResult.valueOf(result_pair.getLeft()), result_pair.getRight().getResult(), "imaResult");
+            },
+            "imaReason", result_pair -> {
+                assertEquals(result_pair.getLeft(), result_pair.getRight().getResultReason(), "imaReason");
+            },
+            "fullAssessmentAvailable", result_pair -> {
+                assertEquals(Boolean.parseBoolean(result_pair.getLeft()), result_pair.getRight().isFullAssessmentPossible(), "fullAssessmentAvailable");
+            },
+            "adjustedIncome", result_pair -> {
+                assertEquals(new BigDecimal(result_pair.getLeft()), result_pair.getRight().getAdjustedIncomeValue(), "adjustedIncome");
```

Comment:
Can remove the brackets for all of these lambdas

Reply:
sorry I don't understand. Which brackets?

Reply:
```
"fullAssessmentAvailable", result_pair -> {
    assertEquals(Boolean.parseBoolean(result_pair.getLeft()), result_pair.getRight().isFullAssessmentPossible(), "fullAssessmentAvailable");
},
```

could be written as:

```
"fullAssessmentAvailable", result_pair -> assertEquals(Boolean.parseBoolean(result_pair.getLeft()), result_pair.getRight().isFullAssessmentPossible(), "fullAssessmentAvailable")
```

That said, you might argue using blocks is more readable. I thought I'd mention it since IDEA keeps popping up with this as a suggestion.